app/models/service_object.rb

```diff
@@ -63,4 +63,14 @@ def validate_key(*keys)
   def default(key, value)
     context.send("#{key}=", value) unless context.send(key).present?
   end
+
+  def something_else
+    if true
```

Comment:
Style/GuardClause: Use a guard clause instead of wrapping the code inside a conditional expression.